test/e2e/performanceprofile/functests/8_reboot/devices.go

```diff
@@ -34,7 +34,7 @@ const (
 
 	// note NO DEFAULT for targetNode - intentionally
 	sriovDeviceResourceNameDefault = "openshift.io/dpdk_nic_1"
-	workloadContainerImageDefault  = "quay.io/fromani/devaccess:2023051001" // FIXME
+	workloadContainerImageDefault  = "quay.io/fromani/devaccess:2023060101" // FIXME
```

Comment:
Thanks for the PR. Are we generally fine using images from private registries ("fromani") in the tests?